model/articles.models.js

```diff
@@ -2,7 +2,14 @@ const db = require('../db/connection')
 
 
 exports.fetchArticlebyId = (id) => {
-    return db.query('SELECT * FROM articles WHERE article_id = $1', [id])
+    return db.query(
+    `SELECT articles.*,
+    COUNT(comment_id) AS comment_count
```

Comment:
add ::int after the closing bracket of (comment_id) this will coerce the count into an integer

Reply:
Damn. Nice one! I 'quietly' changed this value in my test to expect a string so that it would pass lol.